content/practices/event-storming-process-modelling/index.md

```diff
@@ -0,0 +1,151 @@
+---
+# remove "draft:true" when ready for final publication 
+draft: false
+
+title: "EventStorming | Process Modelling"
+linkTitle: "EventStorming | Process Modelling - Will Appear in Links"
+description: "A workshop for facilitating discussions around an outcome based view of a process or workflow, with flexible levels of detail and complexity."
+# Note: remove any tags that are not relevant.
+tags: ["Scoping", "Kickoff", "Discovery", "Framing", "Inception", "Transition", "Modernization", "Delivery"]
+length: "1-2 hours (Lite 30 mins)"
+participants: "Business stakeholders, business analysts, executives, domain experts, core team, developers, architects, team leads"
+# custom "cover" image example: "boris/boris.png"
+image: "default-cover.png" 
+lastmod: "2021-05-21"
+why: 
+- Process Modelling can model complex processes that require discussions between engineers and SMEs and help generate a backlog of work
+- Process modelling can be a quick way to visualize an algorithmic flow or pipeline for use in architectural diagrams and other modelling tools
+- Process Modelling can help compare/hypothesise processes to validate proposed solutions
+when:
+- When discovering a high level business process
+- When discovering a pipeline or workflow
+- When validating process changes that might help solve a problem
+what:
+- "Large wall or digital collaboration space like [Miro](https://miro.com/)"
+- 5+ different colored sticky notes (when using this in conjunction with EventStorming and/or BORIS + SNAPe keep consistency in colors)
+- Sharpies
+
+---
+## How to Use this Method
+This is formally a grammar/notation for modelling Processes and Software, that has evolved from the application of [EventStorming](https://www.eventstorming.com/) by its founder Alberto Brandolini. 
+Traditionally it was indistinguishable from `EventStorming`, however its evolution has lead to Process Modelling becoming its own phase in the `EventStorming` methodology.
+
+While the formal method is best used for identifying large complex high level processes, the general principals and grammar of Process Modelling have been adopted successfully to discover or validate workflows and pipelines at much smaller scales and through existing systems.
+
+The output from these workshops can be used to discover, validate or design a system that supports an end to end business process. The information can be used as an algorithmic model of the problem space, and other forms of design can be derived from it; such as an Event + Domain Driven microservice architecture, a simple Pipeline or even design a `CQRS` and Event Sourcing pattern. Typically there will be a follow on exercise to rationalize the information and transform it from an algorithmic model to a more suitable architectural model.
```

Comment:
This practice is primarily about **Process Modelling** layer, not **Software Modelling** layer (which ultimately is just the addition of aggregates). 
**Process Modelling** can also be used in "Exploration". This can be the exploration of either existing or new systems.
In the DDD-Crew GitHub repo they talk about Aggregates and how you can define them. This would be akin to adding the **Software Modelling** layer and filling out the [aggregate design canvas](https://github.com/ddd-crew/aggregate-design-canvas), or in our case SNAP.
In fact this is exactly how I see it fitting into SWIFT. First **EventStorming** (or **Bigger Picture** or as I like to call it **Storming**) as you normally do, then **Process modelling** as Alberto would do, then **Software Modelling/Candidate Selection**. From there you add stories build Boris and add SNAP diagrams.
Thanks for sending me the link, I can actually use this as a way to explain how it might be used in a DDD Starter Modelling Process.
@joemoore @Maarc  Do you think this would be good to add here?